### PR TITLE
Feature/kah 98

### PR DESCRIPTION
**Pull request by ChBernat**
Positive:
- Removed most of the setting endpoint's functionality since it is not being used anymore.
- Removed entire env folder and all its functionality, moved it to environmental variables
- Lint'ed
- Slightly reorganized main entry file for better readability and easier management
- Prepared better bootstrapping of all variables
- Divided variables into globals and env.

Negative:
- I have set up my email in git's config wrongly hence 3 of my commits are not assigned to my GitHub account. It looks bad but I do not see a point in rewriting history at this point.

Enjoy.

### CODE REVIEW

File: server/.env.example [hunk 1]
```diff
@@ -0,0 +1,7 @@
+UPLOAD_PATH=uploads
+INTERVAL_TIME=5000
+RETRIES=3
+PAGE_SIZE=15
+DB_URI=mongodb://foouser:foopwd@localhost:27017/kaholo
```
Comment (theflexbird) [inline server/.env.example]: Put to example `DB_URI` without user and password

File: server/helpers/setup-env.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    require('dotenv').config();    
+}
+
+const path = require("path");
+const packageJson = require("../package.json");
+
+global.kaholo = {
+    VERSION: packageJson.version,
+    KEY_PATH: path.join(__dirname, ".khkey"),
+    STATIC_CDN: path.join(path.dirname(path.dirname(__dirname)), "static_cdn")
```
Comment (theflexbird) [inline server/helpers/setup-env.js]: Why you have two times `path.dirname`?

File: server/api/tests/settings.test.js [hunk 3]
```diff
@@ -40,55 +13,13 @@ describe("Settings tests", () => {
...
-      });
-    });
   });
 
   describe("POST api/settings/db", () => {
     it(`should respond with 204 after successful connection to given db uri`, () => {
       return request(app)
         .post("/api/settings/db")
-        .send({ uri: originalConfig.dbURI })
+        .send({ uri: "mongodb://127.0.0.1:27017/kaholo" })
```
Comment (theflexbird) [inline server/api/tests/settings.test.js]: Why you used here URI like that? I think it should be taken from `.env`.

File: server/helpers/setup-env.js (hunk 2 above)
Comment (DavidIzaac) [reply server/helpers/setup-env.js]: static_cdn is located in the root folder (PM-Server/ and not in PM-Server/server)

**Pull request merged**